src/components/sideNavigation/tests/SideNavigationLoadingItem.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import { shallow, ReactWrapper, mount } from 'enzyme';
+import { SideNavigationLoadingItem, ISideNavLoadingItemProps } from '../SideNavigationLoadingItem';
+
+describe('<SideNavigationLoadingItem />', () => {
+  let wrapper: ReactWrapper<ISideNavLoadingItemProps, any>;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <SideNavigationLoadingItem />,
+      { attachTo: document.getElementById('App') }
+    );
+  });
+
+  afterEach(() => {
+    wrapper.unmount();
+    wrapper.detach();
+  });
+
+  it('should render a <SideNavigationLoadingItem /> with mod-sub-navigation-left-margin', () => {
+    let container = wrapper.find('div').first();
```

Comment:
Prefer `const` over `let`. https://github.com/coveo/typescript-cloudplatform#references--prefer-const

There is also a linter rule for that (for another time).